src/search/FindInFiles.js

```diff
@@ -417,15 +418,26 @@ define(function (require, exports, module) {
         }
         
         var scopeName = searchModel.scope ? searchModel.scope.fullPath : ProjectManager.getProjectRoot().fullPath,
-            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query);
+            perfTimer = PerfUtils.markStart("FindIn: " + scopeName + " - " + queryInfo.query),
+            searchDeferred = new $.Deferred();
         
-        return candidateFilesPromise
+        candidateFilesPromise
             .then(function (fileListResult) {
                 // Filter out files/folders that match user's current exclusion filter
                 fileListResult = FileFilters.filterFileList(filter, fileListResult);
+                fileListResult = FindUtils.getSortedFiles(fileListResult, function (file) { return file.fullPath; }, FindUtils.getOpenFilePath());
```

Comment:
@TomMalbran @nethip Note that it only takes a long time the first time, so I suspect it's causing the project file structure (?) to get loaded into memory. It would be nice if this process could be started when Find in Files dialog is invoked, but it would probably have to be refactored to be done incrementally so it doesn't not prevent smooth typing in the dialog.

I also thought about showing results panel earlier (to show that we're doing something), but it currently doesn't get displayed if there are no results. We'd have either dismiss it after displaying it (yuck!) or change it so panel is always displayed whether or not there are results (which means more UI to design).